mailpoet/lib/AdminPages/Pages/Newsletters.php

```diff
@@ -109,6 +109,7 @@ public function render() {
       ->format(DateTime::DEFAULT_DATE_FORMAT);
     $data['current_time'] = $dateTime->getCurrentTime();
     $data['current_date_time'] = $dateTime->getCurrentDateTime()->format(DateTime::DEFAULT_DATE_TIME_FORMAT);
+    $data['server_timezone_in_minutes'] = $dateTime->getServerTimezoneInMinutes();
```

Comment:
Because `MailPoet.Date.adjustForTimezoneDifference()` can be used anywhere in the application, I would suggest we move this variable to a global place e.g [layout](https://github.com/mailpoet/mailpoet/blob/6e66bf1f4b77db32ae6fe260e1842303925f7d7d/mailpoet/views/layout.html) or anywhere else that works.

Reply:
@triple0t nice suggestion 😉 Fixed in https://github.com/mailpoet/mailpoet/pull/5650/commits/c8afc972a25316deea1cb330263c635f22de68bd